backend/.env

```
@@ -1,6 +1,5 @@
 DB_PASSWORD=D1ymf8wyQEGthFR1E9xhCq
 DB_USER=LTIdbUser
 DB_NAME=LTIdb
-DB_PORT=5432
-DATABASE_URL="postgresql://${DB_USER}:${DB_PASSWORD}@localhost:${DB_PORT}/${DB_NAME}"
-
+DB_PORT=5441
+DATABASE_URL="postgresql://LTIdbUser:D1ymf8wyQEGthFR1E9xhCq@localhost:5441/LTIdb"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)